addons/account/views/account_payment_view.xml

```diff
@@ -234,7 +234,7 @@
             <field name="name">Payments</field>
             <field name="res_model">account.payment</field>
             <field name="view_mode">tree,kanban,form,graph</field>
-            <field name="context">{'default_payment_type': 'inbound', 'default_partner_type': 'customer', 'search_default_inbound_filter': 1}</field>
```

Comment:
Why remove this default value? The removed fields were `customer` and `supplier` on the partner model, not `partner_type` of a payment.

BTW, the context key `res_partner_search_mode` could be added to show top customers or suppliers on top when searching a partner by name.

```xml
<field name="context">{
    'default_payment_type': 'inbound',
    'default_partner_type': 'customer',
    'search_default_inbound_filter': 1,
    'res_partner_search_mode': 'customer',
}</field>
```